lib/puppet/bindings.rb

```diff
@@ -129,7 +129,7 @@ def [](name)
     end
 
     def has_name?(name)
-      @named_bindings.has_key?
+      @named_bindings.has_key?(name)
```

Comment:
As I understand it, this code is slated for removal in Puppet 5.0.0 and merging this at this point just makes that work having to jump through one extra hoop.